controllers/network_controller.go

```diff
@@ -293,6 +294,13 @@ func (r *NetworkReconciler) reconcilePeeredVNIs(ctx context.Context, log logr.Lo
 	missing := r.setDifference(mbPeerVnis, specPeerVnis)
 	added := r.setDifference(specPeerVnis, mbPeerVnis)
 
+	statusPeerVnis := map[uint32]metalnetv1alpha1.NetworkPeeringState{}
```

Comment:
I would call this variable `networkPeeringState`.

Reply:
Done changed variable name